Sources/Converters.swift

```diff
@@ -347,7 +348,24 @@ class AddConverter: NodeConverter {
     }
 }
 
-class MulConverter: NodeConverter {
+@available(iOS 11.0, macOS 10.13.0, tvOS 11.0, *)
+final class SubConverter: NodeConverter {
+    func convert(in graph: ONNXGraph, node: Onnx_NodeProto) throws {
+        guard
+            let input1 = graph.output(name: node.input[0]),
+            let input2 = graph.output(name: node.input[1]),
+            let inputShape = graph.shape(output: node.input[0]),
+            node.input.count == 2
```

Comment:
Shouldn't we check `.count == 2` before accessing the elements of `node.input` by index?